src/components/lib/VTextField/VTextField.vue

```diff
@@ -195,6 +205,11 @@ input::placeholder {
   color: var(--text-grey);
 }
 
+.form-group[data-disabled='true'],
+input:disabled {
```

Comment:
I recommend using `class` names for styling unless it is a global style.

you thoughts @ajithranka

Reply:
Once we make clean up the CSS, we can enforce this with a lint rule. It's probably okay for now. :-)